project/addons/godot-xr-tools/objects/keyboard/virtual_keyboard_2d.gd

```diff
@@ -108,108 +109,106 @@ func _update_visible():
 
 
 func _on_assumption_pressed():
-	selectArray = $"../../CharacterBody3D".selectArray
-	selectionCount = $"../../CharacterBody3D".selectionCount
+	selectArray = player.selectArray
+	selectionCount = player.selectionCount
 	if selectionCount == 1:
 		selectArray[0].setJustification("assume", "Assume")
```

Comment:
If you're going to use this block of code for every single function in this file, you should abstract it into a helper.